bike_rent/views/website_sale_templates.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<odoo>
+
+<template id="website_sale_product_inherit_bike_rent" inherit_id="website_sale.product">
+    <xpath expr="//section" position="inside">
+        <table class="table table-striped" t-if="product_variant.is_bike">
+            <thead>
+                <tr>
+                    <th scope="col">Manufacturer</th>
+                    <th scope="col">Name</th>
+                    <th scope="col">Model</th>
+                </tr>
+            </thead>
+            <tbody>
+                <t t-foreach="product_variant.rented_bike_ids" t-as="product_line">
+                    <tr t-if="product_line.is_bike">
```

Comment:
If I add non-bike products to `rented_bike_ids` I will get an empty table with a header

Reply:
I've already tested it out, and that was not the case. Let's check this on Friday.

Also - regarding all other cases I've not commented back - either I've changed them as advised, or I'd like to discuss them more on Friday before making changes.

As always - very much appreciate your time!